aliyun/src/main/java/org/apache/iceberg/aliyun/oss/OSSInputFile.java

```diff
@@ -24,17 +24,18 @@
 import org.apache.iceberg.exceptions.ValidationException;
 import org.apache.iceberg.io.InputFile;
 import org.apache.iceberg.io.SeekableInputStream;
+import org.apache.iceberg.metrics.MetricsContext;
 
-public class OSSInputFile extends BaseOSSFile implements InputFile {
+class OSSInputFile extends BaseOSSFile implements InputFile {
```

Comment:
This is technically a breaking change. I think we should roll it back in case anyone references this file. We can deprecate the class so that we can make this change in a release or two.

Reply:
I have rolled back the change.